pkg/discovery/globalnet/globalnet.go

```diff
@@ -32,6 +32,7 @@ type GlobalNetwork struct {
 	ServiceCIDRs []string
 	ClusterCIDRs []string
 	ClusterId    string
+	//Cidrlist     []string
```

Comment:
I had same idea but using a custom CidrType and using switch case. Something like this:
https://play.golang.org/p/cKj4srS3dnl